Flow.Launcher/Flow.Launcher.csproj

```diff
@@ -116,14 +116,4 @@
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
     <Exec Command="taskkill /f /fi &quot;IMAGENAME eq Flow.Launcher.exe&quot;" />
   </Target>
-
-  <Target Name="RemoveDuplicateAnalyzers" BeforeTargets="CoreCompile">
-    <!-- Work around https://github.com/dotnet/wpf/issues/6792 -->
```

Comment:
This work around is no longer required?

Reply:
I think so. It should have been fixed in the latest version.